pythran/pythonic/types/slice.hpp

```diff
@@ -1,6 +1,12 @@
 #ifndef PYTHONIC_TYPES_SLICE_HPP
 #define PYTHONIC_TYPES_SLICE_HPP
 
+#ifdef PYPY_VERSION
```

Comment:
Exactly what I was looking for, one final step: remove that and just use `!defined(PYPY_VERSION)` below :-)